### DISCUSSION TITLE
OpenROAD-flow-scripts 16 built-in sample designs (platform : nangate45)

### DISCUSSION THREAD

**Discussion by srheo-git** [The-OpenROAD-Project/OpenROAD-flow-scripts#2238] [Q&A]
Recently, I'm studying how to use 'OpenROAD-flow-scripts' using 16 built-in example designs.

designs/nangate45/aes
designs/nangate45/ariane133
designs/nangate45/ariane136
designs/nangate45/black_parrot
designs/nangate45/bp_be_top
designs/nangate45/bp_fe_top
designs/nangate45/bp_multi_top
designs/nangate45/bp_quad
designs/nangate45/dynamic_node
designs/nangate45/gcd
designs/nangate45/ibex
designs/nangate45/jpeg
designs/nangate45/mempool_group
designs/nangate45/swerv
designs/nangate45/swerv_wrapper
designs/nangate45/tinyRocket

But, two of them are not running properly (designs/nangate45/bp_quad, designs/nangate45/mempool_group).

Could you check these two examples? and It would be grateful if you give me how to run these two examples properly.

Thank you very much for your time.

**Comment by maliberty** [comment]
Those two were in development by someone who has left the project.  They are not in the CI.  Is there a reason you need to use them?
